src/pages/LoginPage/LoginPage.css

```diff
@@ -0,0 +1,69 @@
+.wrap {
+  display: flex;
+  gap: 160px;
+}
+
+.imgHolder {
+  position: relative;
+  padding: 160px 0 160px 145px;
+  display: flex;
+}
+
+.imgLogo {
+  z-index: 2;
+  height: 132px;
+  border-radius: 100%;
+  margin-right: -65px;
+  margin-top: -66px;
+}
+
+.img {
```

Comment:
Most of these CSS classes need better naming, especially when using same name as HTML tag here. Good way to organise naming descriptively is to prefix it with function that they support e.g. .loginPageImage, .loginPageLogo, .loginPageContainer. loginPageContent etc